src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace Test
+{
+    [Trait("category", "outerloop")]
+    public partial class ParallelQueryCombinationTests
+    {
+        [Theory]
+        [MemberData("UnaryOperators")]
+        [MemberData("BinaryOperators")]
+        public static void Aggregate_OperationCancelledException(LabelledOperation source, LabelledOperation operation)
+        {
+            CancellationTokenSource cs = new CancellationTokenSource();
```

Comment:
...Interestingly, it turns out that It still compares equal.  That feels slightly worrying, actually.
Now I'm wondering if I should go back to the old version and change it to check for reference equals(?)...